content/handbook/support/workflows/account_verification.md

```
@@ -0,0 +1,121 @@
+upstream_path: /handbook/support/workflows/account_verification/
+upstream_sha: 0353e616a41b1d1664a95cc83c80b01f990a912f
+translated_at: "2026-05-08T19:23:50Z"
+translator: claude
+stale: false
+---
+
+## 概要
+
+このワークフローは、ユーザーがアカウント検証を提供する方法をカバーしています。あらゆる [アカウント変更](../workflows/account_changes.md) に使用すべきです。
+
+アカウントアクションは、以下の[ワークフロー](#workflow)が成功した場合にのみ完了できます。
```

markdownlint-cli2 (0.22.1)

[warning] Link fragments should be valid

(MD051, link-fragments)